enclave/src/main/java/org/hyperledger/besu/enclave/GoQuorumEnclave.java

```diff
@@ -58,6 +59,15 @@ public SendResponse send(
         (statusCode, body) -> handleJsonResponse(statusCode, body, SendResponse.class, 201));
   }
 
+  public SendResponse sendSignedTransaction(final byte[] hash, final List<String> privateFor) {
```

Comment:
We have stopped using `hash` to name the transaction lookup id, haven't we? What about renaming it to `txLookupId` or something like that?

Reply:
That is a good idea. I've named it hash, because that name is used in json ...